src/vivarium/framework/resource.py

```diff
@@ -30,7 +30,7 @@ class ResourceError(VivariumError):
     pass
 
 
-RESOURCE_TYPES = {'value', 'value_source', 'value_modifier', 'column', 'stream'}
+RESOURCE_TYPES = {'value', 'value_source', 'missing_value_source', 'value_modifier', 'column', 'stream'}
```

Comment:
To use later in graph display. We allow missing value sources for pipelines that never get called.

Reply:
as in the source of the pipeline is "missing value" ? Is that what that means?

Reply:
This will be a pipeline with no source. Which we allow, I just want to visually represent.